src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -490,9 +494,16 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
      * @param {Function} successCallback - the success callback
      * @param {Function} errorCallback - the error callback
      * @param {string} searchStr - the search string
+     * @param {Object} [options]
```

Comment:
This is the important change - we added a query param for fetching file collaborators that are groups. `approverSelector` uses the query param behind a feature flip; `mentionSelector` does not use it.